### DISCUSSION TITLE
about install dlib

### DISCUSSION THREAD

**Discussion by UltramanDaika** [Tommy-Bie/Face-key-point-detection-and-synthetic-mask-generation#1] [Q&A] [answered]
When I pip install dlib-19.8.1-cp36-cp36m-win_amd64.whl, it was reported the file does not exist.

**Comment by Tommy-Bie** [comment, marked as answer]
The file dlib-19.8.1-cp36-cp36m-win_amd64.whl has been given, and you can go to the directory where this file is located and run pip install.
